baleen/export.py

```diff
@@ -169,60 +179,98 @@ def feedinfo(self, path):
         with open(path, 'w') as f:
             f.write(feeds.to_json(indent=2))
 
-    def export(self, root=None, categories=None, level=SAFE):
+    def export(self, root=None, categories=None, scheme=JSON, level=SAFE):
         """
-        Runs the export of the posts to disk.
+        Export all posts in categories to disk.
         """
-        self.root = root or self.root
+
+        root = root or self.root
+        categories = categories or self.categories
+
+        scheme = scheme or self.scheme
+        level = level or self.sanitize_level
+        if not self.valid_scheme(scheme):
+            raise ExportError(INVALID_EXPORT_SCHEME.format(scheme))
+
+        if not self.valid_sanitize_level(level):
+            raise ExportError(INVALID_SANITIZE_LEVEL.format(level))
+
+        self.initialize_export_directory(root)
+        category_filepaths = self.initialize_category_dirs(base_dir=root,    
```

Comment:
From what I understand - this will create all the subdirectories in the exporter. If there are no posts with an associated category, this means that an empty directory will be created on export, right?